src/uvw/stream.hpp

```diff
@@ -154,9 +154,9 @@ class StreamHandle: public Handle<T, U> {
     }
 
 public:
-#ifdef _WIN32
-    StreamHandle(ConstructorAccess ca, std::shared_ptr<Loop> ref)
-        : Handle{ca, std::move(ref)}
+#ifdef _MSC_VER
+    StreamHandle(typename Handle<T, U>::ConstructorAccess ca, std::shared_ptr<Loop> ref)
+        : Handle<T, U>{ca, std::move(ref)}
```

Comment:
These lines (158-159) should not need to be changed - unless you're getting an explicit error from MSVC. It should be able to deduce these expansions implicitly.